[Docs] Adding Resources, Web instance, and guides pages on Brazilian Portuguese

fellyph opened the pull request:
## Motivation for the change, related issues

This pull request is part of issue #2306, including the translations for the pages: 
- Resources
- Web Instance
- Playground for Theme developers
- Playground for Plugin developers
- IOS app
- Providing content
- Guides home page

adamziel commented:
It seems like there are some build failures

adamziel commented:
Looking at the CI errors, it says:

```
  SyntaxError: Expected double-quoted property name in JSON at position 47
      at JSON.parse (<anonymous>)
      at server.bundle.js:23322:1061
      at Array.map (<anonymous>)
      at parseExamples (server.bundle.js:23322:1025)
      at getBlueprintStepDetails (server.bundle.js:23322:512)
      at getStepAPI (server.bundle.js:23313:408)
      at server.bundle.js:23506:16
      at Array.map (<anonymous>)
      at _createMdxContent (server.bundle.js:23505:32)
      at MDXContent (server.bundle.js:23529:8)
  {
  		"step": "runPHP",
  		"code": "<?php require_once 'wordpress/wp-load.php';
  		wp_insert_post(array('post_title' => 'wp-load.php required for WP
  		functionality', 'post_status' => 'publish')); ?>"
  }
  SyntaxError: Bad control character in string literal in JSON at position 76
      at JSON.parse (<anonymous>)
      at server.bundle.js:23322:1061
      at Array.map (<anonymous>)
      at parseExamples (server.bundle.js:23322:1025)
      at getBlueprintStepDetails (server.bundle.js:23322:512)
      at getStepAPI (server.bundle.js:23313:408)
      at server.bundle.js:23506:16
      at Array.map (<anonymous>)
      at _createMdxContent (server.bundle.js:23505:32)
      at MDXContent (server.bundle.js:23529:8)
```

Which doesn't seem to come from this PR. I wonder if the documentation build job just got triggered for this PR because NX detected a change in the doc package, and it stumbled upon a problem introduced somewhere else.

The issue seems to be in the `packages/playground/blueprints/src/lib/steps/run-php.ts` file:

<img width="1716" height="824" alt="CleanShot 2025-07-30 at 01 33 42@2x" src="https://github.com/user-attachments/assets/d6a84992-d78c-418f-8843-3aa21f568f78" />

Removing the newlines should solve it. It might trigger an eslint error for long comments – it's fine to disable that rule for this comment (via some variant of `eslint-disable` comments)

fellyph commented:
@adamziel, those issues are already in the documentation, because it is caused by the auto-generated API reference, I just ignored them. But I can open an issue to fix it.

adamziel commented:
All good, I went ahead and fixed them today. It would be useful to understand how they snuck in, it seems our CI is not as watertight as it might have seen. Either way we're good to proceed here

adamziel commented:
Ctually there's one more build error that seems to be specific to this change - see the CI report

fellyph commented:
@adamziel, I have updated the branch. The web instance had an outdated version, which was breaking the build.
I have updated and included more pages, and the build process is passing now. 

<img width="1246" height="425" alt="Screenshot 2025-08-05 at 11 53 31" src="https://github.com/user-attachments/assets/eb92f98a-ebc4-4b73-8e50-eaa921a4bc8a" />

adamziel commented:
Nice, thank you @fellyph! I've added the labels and merged
